tools/container.bzl

```diff
@@ -1,11 +1,51 @@
-load("@io_bazel_rules_docker//container:container.bzl", "container_push")
+load("@aspect_bazel_lib//lib:transitions.bzl", "platform_transition_filegroup")
+load("@rules_oci//oci:defs.bzl", "oci_push", "oci_image", "oci_image_index")
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+
+def multiarch_go_image(name, binary):
+    """Create a container image with two variants of the given go_binary target.
+    
+    Args:
+        name: resulting oci_image_index target
+        binary: label of a go_binary target; it may be transitioned to another architecture
+    """
+    images = []
+    tar_target = "_{}.tar".format(name)
+    image_target = "_{}.image".format(name)
+
+    pkg_tar(
+        name = tar_target,
+        srcs = [binary],
+        include_runfiles = True,
+        package_dir = "app",
+    )
+
+    oci_image(
+        name = image_target,
+        base = "@distroless_static",
```

Comment:
Wouldn't this expand to /app/:bb_storage? Notice the colon.

Reply:
yes, just added a removeprefix on that, I noticed as well

Reply:
Can't we use Label(binary).name here instead?

Reply:
not quite - in the macro context the string isn't fully qualified:

`Error in Label: Illegal absolute label syntax: invalid label ':bb_copy': absolute label must begin with '@' or '//'`

Reply:
What about https://bazel.build/rules/lib/toplevel/native.html#package_relative_label ?

Reply:
Yeah I think that's what we want, but it was introduced in Bazel 6.1.0 https://github.com/bazelbuild/bazel/commit/b859571804f2095caaf018b172b59c90f185fd51 and this repo currently uses Bazel 6.0.0

Reply:
Then let's try upgrading to 6.1.0 or later.

Reply:
pushed a commit to upgrade to Bazel 6.1.0, guessing that's what you'd prefer